API-Automation/test-suites/pet/get_request.spec.js

```diff
@@ -0,0 +1,19 @@
+import { findPetById, findPetByStatus } from "../../tasks/api-tasks.js";
+
+
+describe('PET - GET API - positive', () => {
+
+    it('TC_10_1 User is able to find the pet by status', async () => {
+        const response = await findPetByStatus();
+        expect(response.status).toBe(200);
+        expect(response.data).toBeDefined();
```

Comment:
And then, you know which payload you used in i.e. beforeAll(), then you can add here also like in post request spec:

expect (_.isEqual(response.data,newPetData));